modules/functional_expansion_tools/include/series/FunctionalBasisInterface.h

```diff
@@ -0,0 +1,180 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#ifndef FUNCTIONALBASISINTERFACE_H
+#define FUNCTIONALBASISINTERFACE_H
+
+// MOOSE includes
+#include "MooseEnum.h"
+#include "MooseError.h"
+#include "MooseTypes.h"
+
+// Shortened typename
+class FunctionalBasisInterface;
+typedef FunctionalBasisInterface FBI;
+
+/**
+ * Passkey for ...SeriesBasisInterface
+ */
+class SBIKey
```

Comment:
I realized it isn't needed. I changed it so that `SingleSeriesBasisInterface` and `CompositeSeriesBasisInterface` both declare `clearBasisEvaluation(...)` as private.